notebooks/notebooks/iclr2022/baseline-attacks.ipynb

```diff
@@ -15,125 +15,405 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        features = path.stem.split("-")
</pre><p>It's a bit hard to follow what this is doing, consider adding a comment giving example path format? e.g:</p><pre class="ql-syntax" spellcheck="false"># path.stem is of format cpN[h]-vN-{bw}-vs-(attack).sgfs, e.g. cp505h-v16-b-vs-edge.sgfs
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>